content/developer/writing_widgets/widget-definition.md

```diff
@@ -141,6 +141,10 @@ fetchUrl: '[manager]/executions?is_system_workflow=false[params]'
      Please note that this can be used both to selectively pick pagination parameter as well as custom parameters (see [fetchParams function](#fetchparams-widget-toolbox)).
       This mode is **exclusive** - parameters not specified explicitly are skipped.
       When using selective param picking (`[params:param_name]`) you can use a pre-defined `gridParams` tag to include all pagination parameters (`_size`, `_offset`, `_sort`) instead of specifying explicitly  each of the three.
```

Comment:
I'm wondering... Should we not include `_search` as one of the default `gridParams`?

Reply:
_search is part of the existing component, it was just enabled by one property.

Changes:
https://github.com/cloudify-cosmo/cloudify-stage/pull/1195/files

Reply:
My question is on the Stage's widget API level, not on the level of this individual change to the documentation :smile: And probably @qooban could answer it

Reply:
@Gelio `_search` is already part of the `gridParams`.

@gregcfy Check `mapGridParamsToParams` method in `cloudify-stage/app/utils/WidgetParamsHandler.js`. 

I originally thought you will just extend a list in: 

> all pagination parameters (`_size`, `_offset`, `_sort`)

within this PR :-)